src/components/Featured.tsx

```diff
@@ -1,23 +1,18 @@
 import React from 'react'
-import {
-  Container,
-  Stack,
-  Flex,
-  Box,
-  Heading,
-  Text,
-  Button,
-  Image,
-  Icon,
-  IconButton,
-  createIcon,
-  IconProps,
-  useColorModeValue,
-} from '@chakra-ui/react'
-import { FEATURED_TEXT, FEATURED_IMAGE } from 'utils/context'
-import { Player } from '@livepeer/react'
+import { Container, Stack, Flex, Box, Heading, Text, Image, IconButton, createIcon } from '@chakra-ui/react'
+import { LIVEPEER_FEATURED_PLAYBACK_ID } from 'utils/config'
+import { FEATURED_TEXT } from 'utils/context'
+import { createReactClient, studioProvider, LivepeerConfig, Player } from '@livepeer/react'
 
-const FEATURED_PLAYBACK_ID = 'c3c7u34h2y47jqpd'
+declare var process: {
+  env: {
+    NEXT_PUBLIC_STUDIO_API_KEY: string
+  }
+}
+
+const client = createReactClient({
+  provider: studioProvider({ apiKey: process.env.NEXT_PUBLIC_STUDIO_API_KEY }),
+})
```

Comment:
Noticed we create a livepeer react client in several files. Might be better to create a single client, export it, and import it wherever we need it

Reply:
Yeah, that makes sense.

Reply:
Agree on that.